src/helpers/CameraHelper.js

```diff
@@ -30,58 +30,58 @@ class CameraHelper extends LineSegments {
 
 		// colors
 
-		const colorFrustum = new Color( 0xffaa00 );
-		const colorCone = new Color( 0xff0000 );
-		const colorUp = new Color( 0x00aaff );
-		const colorTarget = new Color( 0xffffff );
-		const colorCross = new Color( 0x333333 );
+		const _colorFrustum = new Color( colorFrustum );
+		const _colorCone = new Color( colorCone );
+		const _colorUp = new Color( colorUp );
+		const _colorTarget = new Color( colorTarget );
+		const _colorCross = new Color( colorCross );
```

Comment:
This naming convention here is ugly, but I'd rather the arguments have the nice names - as they are user-facing